cmake/FindOVS.cmake

```diff
@@ -0,0 +1,131 @@
+# FindOVS.cmake - find Open vSwitch package.
+#
+# Copyright 2022-2023 Intel Corporation
+# SPDX-License-Identifier: Apache 2.0
+#
+
+#-----------------------------------------------------------------------
+# Use pkg-config to search for the modules.
+#-----------------------------------------------------------------------
+find_package(PkgConfig)
+pkg_check_modules(PC_OVS QUIET libopenvswitch)
+
+#-----------------------------------------------------------------------
+# Find libraries and include directories.
+#-----------------------------------------------------------------------
+find_path(OVS_INCLUDE_DIR
+    NAMES "openvswitch/version.h"
+    PATHS ${PC_OVS_INCLUDE_DIRS}
+)
+
+find_library(OVS_LIBRARY
+    NAMES openvswitch
+    PATHS ${PC_OVS_LIBRARY_DIRS}
+)
+
+find_library(OFPROTO_LIBRARY
+    names ofproto
+    PATHS ${PC_OVS_LIBRARY_DIRS}
+)
+
+find_library(OVSDB_LIBRARY
+    names ovsdb
+    PATHS ${PC_OVS_LIBRARY_DIRS}
+)
+
+find_library(SFLOW_LIBRARY
+    names sflow
```

Comment:
Valid option according to cmake documentation - https://cmake.org/cmake/help/latest/command/find_library.html is uppercase **NAMES**
<img width="619" alt="image" src="https://github.com/ipdk-io/networking-recipe/assets/22621220/1f66d0c3-68bb-4f25-a845-2127ba25056d">

Also, I see lowercase **names** is not encoded in red indicating .cmake is not detecting it as valid option